javatests/arcs/android/storage/database/DatabaseImplTest.kt

```diff
@@ -418,7 +419,8 @@ class DatabaseImplTest {
                     "float" to FieldType.Float,
                     "double" to FieldType.Double,
                     "txtlst" to FieldType.ListOf(FieldType.Text),
-                    "lnglst" to FieldType.ListOf(FieldType.Long)
+                    "lnglst" to FieldType.ListOf(FieldType.Long),
+                    "bigint" to FieldType.BigInt
```

Comment:
`bigintlst` as well?

Reply:
it's not really necessary; that codepath should be covered by txtlst. I can add it if you want.